go.mod

```diff
@@ -65,10 +65,10 @@ require (
 	github.com/vbauerster/mpb/v8 v8.10.2
 	github.com/vishvananda/netlink v1.3.1
 	go.etcd.io/bbolt v1.4.3
-	go.podman.io/common v0.65.1-0.20250925174758-4cf0ff781bfc
+	go.podman.io/common v0.65.1-0.20251016133615-aa970d2c7532
 	go.podman.io/image/v5 v5.37.0
 	go.podman.io/storage v1.60.0
```

Comment:
please vendor all modules at once so we don't introduce weird in between things

Reply:
done